pkg/workflow/argo/fetch.go

```diff
@@ -133,9 +133,9 @@ func (w *Workflow) Fetch(req *pb.FetchRequest) (*pb.FetchResponse, error) {
 		// TODO(yancey1989): add duration time for the eoeResponse
 		// eoe just used to simplify the client code which can be consistent with non-argo mode.
 		if isPodFailed(pod) {
-			logger.Errorf("workflowFailed, spent:%d", time.Now().Second()-wf.CreationTimestamp.Second())
```

Comment:
I believed that we should log this message in sqlflowserver log files, we can analyze these message to find how many SQLFlow jobs fault. And maybe a better way is to disable output the logs on CI, it's not helpful to diagnostic error.

Reply:
I agree with what you said. So? It would be helpful to tell the conclusion in a recommendation, so the receiver, in this case @weiguoz , knows what to do.

Reply:
the logs are steel exists:
``` text
time="2020-05-20T08:58:08Z" level=info msg="phase:Running" event=fetch jobID=sqlflow-tbff4 requestID=068a386a-9a78-11ea-8aa7-42010a1e0045 stepID=sqlflow-tbff4-2595358660
time="2020-05-20T08:58:12Z" level=info msg="phase:Running" event=fetch jobID=sqlflow-tbff4 requestID=091c264e-9a78-11ea-8aa7-42010a1e0045 stepID=sqlflow-tbff4-2595358660
time="2020-05-20T08:58:17Z" level=info msg="phase:Running" event=fetch jobID=sqlflow-tbff4 requestID=0ba2d3ff-9a78-11ea-8aa7-42010a1e0045 stepID=sqlflow-tbff4-2595358660
time="2020-05-20T08:58:17Z" level=info msg="workflowSucceed, spent:-21" event=fetch jobID=sqlflow-tbff4 requestID=0ba2d3ff-9a78-11ea-8aa7-42010a1e0045 stepID=sqlflow-tbff4-2595358660
time="2020-05-20T08:58:21Z" level=info msg="phase:Succeeded" event=fetch jobID=sqlflow-tbff4 requestID=0e26f8cb-9a78-11ea-8aa7-42010a1e0045 stepID=sqlflow-tbff4-1588554425
time="2020-05-20T08:58:21Z" level=info msg="workflowSucceed, spent:-17" event=fetch jobID=sqlflow-tbff4 requestID=0e26f8cb-9a78-11ea-8aa7-42010a1e0045 stepID=sqlflow-tbff4-1588554425
```

I recommended disabling these logs on CI, just like `> /dev/null`, we can do that in Go:

``` golang
var Discard io.Writer = devNull(0)
```

Reply:
SQLFlow still logs the error message, with [more error details](https://github.com/sql-machine-learning/sqlflow/pull/2317/files#diff-fe9667b19cdb8812096441badccf1678R137) for diagnostic. 

Also, we'are able to [specify the log file](https://github.com/sql-machine-learning/sqlflow/blob/fae853e1df/cmd/sqlflowserver/main.go#L85) in the command line. If we ignore the [logPath](https://github.com/sql-machine-learning/sqlflow/blob/fae853e1df/cmd/sqlflowserver/main.go#L85), the log will be thrown out to `stdout/stderr` as the default behavior. However, it seems we have no chance to specify the `logPath` in CI.